x-pack/plugins/features/server/plugin.ts

```diff
@@ -40,22 +40,24 @@ import {
  */
 export interface PluginSetupContract {
   registerKibanaFeature(feature: KibanaFeatureConfig): void;
+
   registerElasticsearchFeature(feature: ElasticsearchFeatureConfig): void;
+
   /**
    * Calling this function during setup will crash Kibana.
    * Use start contract instead.
    * @deprecated
    * @removeBy 8.8.0
```

Comment:
Not directly related to this PR: Are we still planning on removing the `get<Kibana/Elasticsearch>Features` APIs? IF not, I think we should remove the deprecated notice.

Reply:
I was planning on taking a look and trying to remove them to see if the CI complains in another PR